### PR TITLE
Add toggle to save and load mounting reset

### PR DESCRIPTION
**Pull request by Louka3000**
Useful for people with suits like my friend GizmoQC ;)
Added a warning for normal users (see below)

SolarXR PR: https://github.com/SlimeVR/SolarXR-Protocol/pull/133

### CODE REVIEW

**Comment by Louka3000** [comment]
slimevr_zC6d6RMCZk

**Comment by Louka3000** [comment]
Merge https://github.com/SlimeVR/SlimeVR-Server/pull/953 before this.

File: gui/src/components/settings/pages/GeneralSettings.tsx [hunk 1]
```diff
@@ -560,7 +564,7 @@ export function GeneralSettings() {
                 step={0.1}
               />
             </div>
-            <div className="flex gap-5 pt-5 md:flex-row flex-col">
+            <div className="flex gap-5 pt-5 pb-7 md:flex-row flex-col">
```
Comment (ImUrX) [inline gui/src/components/settings/pages/GeneralSettings.tsx:567]: modify the actual div that you are making please
```suggestion
            <div className="flex gap-5 pt-5 md:flex-row flex-col">
```

File: gui/src/components/settings/pages/GeneralSettings.tsx [hunk 2]
```diff
@@ -572,6 +576,28 @@ export function GeneralSettings() {
...
+              )}
+            </Typography>
+            <div className="flex flex-col pt-2 pb-3">
+              <Localized
+                id="settings-general-tracker_mechanics-save_mounting_reset-description"
+                elems={{ b: <b></b> }}
+              >
+                <Typography color="secondary"></Typography>
+              </Localized>
+            </div>
```
Comment (ImUrX) [inline gui/src/components/settings/pages/GeneralSettings.tsx:591]:
```suggestion
            <div className="flex flex-col pt-2 pb-3">
              <Typography bold>
                {l10n.getString(
                  'settings-general-tracker_mechanics-save_mounting_reset'
                )}
              </Typography>
              <Localized
                id="settings-general-tracker_mechanics-save_mounting_reset-description"
                elems={{ b: <b></b> }}
              >
                <Typography color="secondary"></Typography>
              </Localized>
            </div>
```

**Review by ImUrX — CHANGES_REQUESTED** [review]
if you want more gap between the title and the description use something like `gap-3`x.

there is merge conflicts, i will review the rest after fixing that

**Review by ImUrX — APPROVED** [review]

**Comment by Louka3000** [comment]
Needs https://github.com/SlimeVR/SolarXR-Protocol/pull/133

**Comment by Louka3000** [comment]
CI will fail because solarxr doesn't have yaw smoothing

**Comment by Louka3000** [comment]
Good for merge (solarxr points to main and CI is gud)

**Review by ButterscotchV — APPROVED** [review]

**Pull request merged**